tests/unit/entityInfo.spec.ts

```diff
@@ -6,43 +6,76 @@ import Vuetify from 'vuetify'
 import { getVuexStore } from '@/store'
 
 // Components
-import { mount } from '@vue/test-utils'
+import { createLocalVue, mount } from '@vue/test-utils'
 import { EntityInfo } from '@/components/common'
 
-Vue.use(Vuetify)
+// Other
+import mockRouter from './MockRouter'
+import VueRouter from 'vue-router'
 
+Vue.use(Vuetify)
 const vuetify = new Vuetify({})
-
-describe.skip('Entity Info component with an NR', () => {
-  let wrapper: any
-  let store: any
+document.body.setAttribute('data-app', 'true')
+
+describe('Entity Info component in a Correction as a named company', () => {
+  let wrapper
+  let store
+
+  const mockFiling = {
+    header: {
+      name: 'incorporationApplication',
+      filingId: 12345
+    },
+    business: {
+      identifier: 'BC1234567',
+      legalType: 'BEN'
+    },
+    incorporationApplication: {
+      contactPoint: {
+        email: 'mock@email.com',
+        phone: '123-456-7890'
+      },
+      nameRequest: {
+        legalType: 'BEN',
+        nrNumber: 'NR 1234567',
+        legalName: 'My Mock Name Inc.'
+      },
+      offices: {},
+      parties: [],
+      shareClasses: [],
+      incorporationAgreement: {}
+    }
+  }
 
   beforeEach(() => {
     store = getVuexStore()
-
-    // Entity type will always be set with or without an NR
-    store.state.stateModel.tombstone.entityType = 'BEN'
-    // Temp Id will always be set with or without an NR
-    store.state.stateModel.tombstone.businessId = 'T1234567'
-    store.state.stateModel.nameRequest.nrNumber = 'NR 1234567'
-    store.state.stateModel.nameRequest.details.approvedName = 'Xyz Ltd.'
-
-    wrapper = mount(EntityInfo, { vuetify, store })
+    let state = store.state.stateModel
+
+    state.tombstone.keycloakRoles = ['staff']
```

Comment:
You can move this common stuff, which only needs to be called once, into a `beforeAll` method.

Nice use of `state` to shorten the assignments BTW 👍 
(It might even be a bit quicker as you don't have to de-reference the store object every time.)

Reply:
nice, good optimization! Updated in next commit.